raigad/src/main/java/com/netflix/raigad/monitoring/HealthMonitor.java

```diff
@@ -111,6 +110,7 @@ else if (clusterHealthStatus.name().equalsIgnoreCase("RED")) {
             logger.warn("failed to load Cluster Health Status", e);
         }
 
+        healthBean.esuponinstance = ElasticsearchProcessMonitor.isElasticsearchStarted() ? 1 : 0;
```

Comment:
This check will always be true. Meaning, here you won't be able to catch if ES is not running, because of Initial Check in the beginning of execute() method.